content/en/docs/tasks/observability/logs/otel-provider/index.md

```diff
@@ -7,6 +7,10 @@ owner: istio/wg-policies-and-telemetry-maintainers
 test: yes
 ---
 
+{{< warning >}}
+The OpenTelemetry provider is available in Istio `v1.13.3` or later.
```

Comment:
On other parts the page and in the title, we say collector. Should this be the same?

Reply:
`OpenTelemetry provider` is a extension provider type in Istio.